framework/src/problems/FEProblemBase.C

```diff
@@ -692,7 +692,7 @@ FEProblemBase::createTagSolutions()
     // ease of use in doing things like copying solutions backwards. We're just storing pointers in
     // the solution states containers so populating the zeroth state does not cost us the memory of
     // a new vector
-    needSolutionState(2, Moose::SolutionIterationType::Nonlinear);
+    needSolutionState(1, Moose::SolutionIterationType::Nonlinear);
```

Comment:
lol what the hell. Nice fix. I wouldn't be surprised if git blame points to me on this one